marshmallow/validate.py

```diff
@@ -107,6 +109,7 @@ class Email(Validator):
         interpolated with `{input}`.
     """
 
+    __slots__ = ('error')
```

Comment:
Actually it's fine ([it's automatically converted to tuple](https://github.com/python/cpython/blob/2.7/Objects/typeobject.c#L2197)). Still a bit inconsistent style tho :-)